### PR TITLE
Add Dockerfile for content-tagger app

### PR DESCRIPTION
**Pull request by adrianclay**
Is required for running publisher under publishing-e2e-tests.

### CODE REVIEW

**Comment by cbaines** [comment]
> Is required for running publisher under publishing-e2e-tests.

You probably want to say Content Tagger, rather than publisher.

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,22 @@
+FROM ruby:2.4.2
+RUN apt-get update -qq && apt-get upgrade -y && apt-get install -y build-essential nodejs && apt-get clean
+
+ENV DATABASE_URL postgresql://postgres@postgres/content_tagger_development
```
Comment (kevindew) [inline Dockerfile]: shall we drop development from this? (since we try keep that portable)

File: db/seeds.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+end
+
+gds_organisation_id = "af07d5a5-df63-4ddc-9383-6a666845ebe9"
+
+User.create!(
+  uid: SecureRandom.uuid,
+  name: "Test user",
+  permissions: ["signin", "GDS Editor", "Tagathon participant"],
+  organisation_content_id: gds_organisation_id,
+)
```
Comment (cbaines) [inline db/seeds.rb]: You're missing a newline here.

Comment (adrianclay) [reply db/seeds.rb]: Doh, new IDE needed reconfiguring.

**Pull request merged**